keyboards/cannonkeys/atlas_alps/readme.md

```diff
@@ -0,0 +1,14 @@
+Cannon Key Atlas_Alps
+===
+
+The Atlas_Alps PCB is an alternate PCB for the Atlas as sold by Cannon Keys. Instead of hotswap, it includes compatibility for Alps and MX switches.
+
+Keyboard Maintainer: Nasp  
+Hardware Supported: Atlas Atmega 32u4   
+Hardware Availability: https://geekhack.org/index.php?topic=106683.msg2975081#msg2975081
```

Comment:
```suggestion
# Cannon Keys Atlas_Alps

The Atlas_Alps PCB is an alternate PCB for the Atlas as sold by Cannon Keys. Instead of hotswap, it includes compatibility for Alps and MX switches.

* Keyboard Maintainer: Nasp
* Hardware Supported: ATmega32U4
* Hardware Availability: https://geekhack.org/index.php?topic=106683.msg2975081#msg2975081
```